### PR TITLE
fix: notification styles

### PR DESCRIPTION
**Pull request by shhdgit**
Show notification when the version not exist.
image

### CODE REVIEW

File: ui/src/main.tsx [hunk 1]
```diff
@@ -30,7 +31,7 @@ ReactDOM.createRoot(document.getElementById('root')!).render(
         <ColorSchemeScript defaultColorScheme="auto" />
         <MantineProvider defaultColorScheme="auto">
           <ModalsProvider>
-            <Notifications position="top-right" />
+            <Notifications position="top-center" zIndex={99999999} />
```
Comment (lucklove) [inline ui/src/main.tsx:34]: Why 99999999?

Comment (shhdgit) [reply ui/src/main.tsx:34]: prevent vision block by the fixed style in the future, like:
image

Comment (shhdgit) [reply ui/src/main.tsx:34]: Notification should be at the top of all elements.

Comment (lucklove) [reply ui/src/main.tsx:34]: But `99999999` is a magic number, should be replaced with project level const var to avoid someone use more `9` than you.

**Review by lucklove — APPROVED** [review]
LGTM

**Pull request merged**